src/components/portfolio/DetailsMainAnalyticsHeatmap.tsx

```diff
@@ -1,13 +1,31 @@
 import React from 'react';
 import Chart from 'react-apexcharts';
-import { useTheme } from '@material-ui/core';
 import { useTranslation } from 'react-i18next';
+import {
+  makeStyles,
+  useTheme,
+  createStyles,
+  Theme,
+} from '@material-ui/core/styles';
 import {
   Correlations,
   NonEmptyPortfolioDetails,
 } from '../../portfolio/APIClient';
 import { roundAxis } from '../../portfolio/Helper';
 
+const useStyles = makeStyles(({ palette }: Theme) =>
+  createStyles({
+    infoBody: {
+      display: 'flex',
+      alignSelf: 'center',
+      width: '100%',
+      justifyContent: 'center',
+      color: palette.primary.contrastText,
+      fontSize: '1.15rem',
+    },
+  })
+);
+
```

Comment:
If it's not too much a bother I think it would look better if the infoBody had more margin top and bottom to fill out the sections a bit. (also alignSelf: 'center' has no impact in this context). I think this change would have to be done for all infoBodies including the dividends section.

```suggestion
const useStyles = makeStyles(({ palette }: Theme) =>
  createStyles({
    infoBody: {
      display: 'flex',
      margin: '15rem 0',
      width: '100%',
      justifyContent: 'center',
      color: palette.primary.contrastText,
      fontSize: '1.15rem',
    },
  })
);

```

Reply:
I added a new css styling class since infobox is already used in dividends for something else, the page should look better now